src/cogs/admin.py

```
@@ -0,0 +1,98 @@
+            return
+
+        # Check if user is linked first
+        db_user = get_user_by_discord(user.id)
+        if not db_user:
+             await interaction.response.send_message(f"❌ {user.mention} is not linked! They must use `/link` first.", ephemeral=True)
+             return
+
+        if add_maintainer(user.id, repo_url):
+            await interaction.response.send_message(f"✅ Added {user.mention} as maintainer for {repo_url}")
+        else:
+            await interaction.response.send_message(f"❌ Failed. Check if they are already a maintainer or if the repo is valid.", ephemeral=True)
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)